Source/Core/InputCommon/ControllerInterface/GameController/GameController.mm

```diff
@@ -0,0 +1,80 @@
+// Copyright 2013 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <random>
+
+#import <objc/message.h>
+
+#include "InputCommon/ControllerInterface/GameController/GameController.h"
+#include "InputCommon/ControllerInterface/GameController/GameControllerDevice.h"
+
+#include "Common/Logging/Log.h"
+
+namespace ciface::GameController
+{
+static char *s_observer="";
+
+void addController(GCController* controller)
+{
+  controller.motion.sensorsActive = true;
+
+  std::string vendorName = std::string([controller.vendorName UTF8String]);
```

Comment:
```suggestion
  std::string vendor_name = std::string([controller.vendorName UTF8String]);
```

Reply:
I don't particularly care but I think this string be const